modules/installation-cis-ibm-cloud.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="installation-cis-ibm-cloud_{context}"]
-= Configuring DNS resolution using Cloud Internet Services
+= Using IBM Cloud Internet Services for DNS resolution
```

Comment:
DNS => CIS ?

Reply:
Note the section above (Configuring DNS resolution). It explains that depending on the type of cluster you are installing determines how you configure DNS resolutions. The steps for CIS are directly below the section you are referencing. [1]

[1] [Using IBM Cloud DNS Services for DNS resolution](https://50561--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-account.html#installation-dns-ibm-cloud_installing-ibm-cloud-account)

Reply:
How about change the title as `Using IBM Cloud Internet Services (CIS) for DNS resolution`